src/Routers/ClassesRouter.js

```diff
@@ -129,6 +129,7 @@ export class ClassesRouter extends PromiseRouter {
     this.route('GET', '/classes/:className', (req) => { return this.handleFind(req); });
     this.route('GET', '/classes/:className/:objectId', (req) => { return this.handleGet(req); });
     this.route('POST', '/classes/:className', (req) => { return this.handleCreate(req); });
+    this.route('PUT', '/classes/:className', (req) => { return this.handleUpdate(req); });
```

Comment:
We should require masterKey here, I believe `this.route(` supports any number of args, this way we can revert all code inside the `src/rest.js`